CMakeLists.txt

```diff
@@ -55,11 +53,11 @@ set(NEXTPVR_HEADERS src/addon.h
                     src/buffers/Seeker.h
                     src/utilities/XMLUtils.h)
 
-SET(DEPLIBS ${p8-platform_LIBRARIES}
-            ${TINYXML2_LIBRARIES})
+SET(DEPLIBS ${TINYXML2_LIBRARIES})
 if(WIN32)
   list(APPEND DEPLIBS ws2_32)
   add_definitions(/D_WINSOCK_DEPRECATED_NO_WARNINGS)
+  add_definitions(-D _WINSOCKAPI_)
```

Comment:
You was faster 😄.

Thought to write:
```cmake
  add_definitions(-D_WINSOCKAPI_ -D_WINSOCK_DEPRECATED_NO_WARNINGS)
```

Think as one line looks better and also is the `/` not direct needed.

**EDIT** Also without space as I thing not set correct.

Reply:
Yes thanks that looks much cleaner